homeassistant/components/unifi/unifi_client.py

```diff
@@ -25,10 +25,12 @@
 CLIENT_BLOCKED = (WIRED_CLIENT_BLOCKED, WIRELESS_CLIENT_BLOCKED)
 CLIENT_UNBLOCKED = (WIRED_CLIENT_UNBLOCKED, WIRELESS_CLIENT_UNBLOCKED)
 WIRED_CLIENT = (WIRED_CLIENT_CONNECTED, WIRED_CLIENT_DISCONNECTED)
+WIRELESS_CLIENT_ROAMRADIO = "EVT_WU_RoamRadio"
 WIRELESS_CLIENT = (
     WIRELESS_CLIENT_CONNECTED,
     WIRELESS_CLIENT_DISCONNECTED,
     WIRELESS_CLIENT_ROAM,
+    WIRELESS_CLIENT_ROAMRADIO,
```

Comment:
I wasn't thinking about it, but should guest events be included here as well?

Reply:
I haven't gotten any feedback that guests should work. So I mainly document it in code in case that changes